### ISSUE TITLE
Feature: Activity with custom calories

### ISSUE THREAD

**Issue by maex-m** [simonoppowa/OpenNutriTracker#70]
> Add generic activity by providing burned calories in kcal

The current approach with information from the _"2011 Compendium of Physical Activities"_ is a great approach.
Nevertheless, some users may already use fitness trackers that provide accurate information about burned calories.

Implementing connections between fitness apps and _OpenNutriTracker_ will probably conflict with privacy principles.
Also the effort of using such APIs would be quite big.

A privacy friendly and low effort solution would therefore be a feature to add activities with custom calorie values.

**Comment by Hooloovoo** [comment]
Yes, or just make it possible to override the looked up value with the value from your e.g. running app.

**Comment by MooreDylanR** [comment]
yes if i do a workout with my watch on it will tell me how many calories are burned, i should be able to enter "600 kcal". currently the only unit is 'minutes' then it does its own calculation of how many calories that is. 

current work around is to lie about minutes until the kcal matches, adding kcal as a unit on the workout would be better.

**Comment by jordan-lee-code** [comment]
Hi @maex-m — confirming this is still outstanding on `develop`.

Code location: `lib/features/add_activity/` has no custom-MET / custom-kcal entry path; the activity list at `lib/core/data/data_source/physical_activity_data_source.dart` is static. Adding this would mean a "Custom activity" type plus a form for direct kcal entry (which would write through a new code path that bypasses the MET × weight × hours computation in `MetCalc`).

Labelled `enhancement`. Note: #33 was closed as duplicate of this issue.

**Comment by jordan-lee-code** [comment]
Merged in #408. Thanks for filing — this is now on `develop` and will ship with the next release.

If you spot something still wrong on a fresh build, please reopen this issue or open a new one with the details and I'll have a look.
